images/linux/toolsets/toolset-1604.json

```diff
@@ -214,6 +214,7 @@
             "buildpack-deps:buster",
             "debian:9",
             "debian:10",
+            "moby/buildkit:latest",
```

Comment:
Ubuntu 16.04 is end of support. No need to update.

Reply:
Done